src/primitives.rs

```diff
@@ -35,3 +35,14 @@ pub fn new_false<'sc>(scope: &mut impl ToLocal<'sc>) -> Local<'sc, Boolean> {
   let ptr = unsafe { v8__False(scope.isolate()) };
   unsafe { scope.to_local(ptr) }.unwrap()
 }
+
+pub fn new_boolean<'sc>(
```

Comment:
Not a big deal of course but can we match the V8 unless there's a good reason not to.
v8 has this function:

```
Local<Boolean> Boolean::New(Isolate* isolate, bool value) {
  return value ? True(isolate) : False(isolate);
}
```

Reply:
Sure, why not

Reply:
No I mean actually exactly the opposite. I don't care how it's implemented.
I meant that the rust api should be v8::Boolean::new.

Or is there a reason to do it differently?